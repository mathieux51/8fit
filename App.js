import React, { Component } from "react"
import { Font } from "expo"
import Root from "./src/containers/Root"

class App extends Component {
  state = {
    fontLoaded: false
  }
  async componentDidMount() {
    await Font.loadAsync({
      "fira-sans-bold": require("./src/assets/fonts/FiraSans-Bold.ttf"),
      "fira-sans-medium": require("./src/assets/fonts/FiraSans-Medium.ttf"),
      "fira-sans-regular": require("./src/assets/fonts/FiraSans-Regular.ttf")
    })
    this.setState({ fontLoaded: true })
  }
  render() {
    return this.state.fontLoaded && <Root />
  }
}

export default App
