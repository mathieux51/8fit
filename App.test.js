import React from "react"
import App from "./App"
import { shallow } from "enzyme"
// import sinon from "sinon"

describe("<App />", () => {
  it("renders correctly", () => {
    expect(shallow(<App />)).toMatchSnapshot()
  })
})
