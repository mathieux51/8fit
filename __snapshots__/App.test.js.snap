// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): false,
  Symbol(enzyme.__nodes__): Array [
    false,
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
